# Python program to handle simple runtime error
#-------------------------------------------
# simple/generic exception handling
a = [1, 2, 3]
try:
    print ("Second element = %d" %(a[1]))
    # Throws error since there are only 3 elements in array
    print ("Fourth element = %d" %(a[3]))
except:
    print ("An error occurred")
    
finally:
    # this block is always executed
    # regardless of exception generation.
    print('This is always executed')
#-------------------------------------------
